src/index.js

```diff
@@ -0,0 +1,170 @@
+"use strict";
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    increaseTempControl: null,
+    decreaseTempControl: null,
+    tempValue: null,
+    landscape: null,
+    headerCityName: null,
+    cityNameInput: null,
+    currentTempButton: null,
+    temp: tempValue.textContent,
+    skySelect: null,
+    sky: null,
+    cityNameReset: null,
+    gardenContent: null,
+};
+
+const loadControls = () => {
+    for (const element in state) {
+        if (element !== 'temp') {
+            state[element] = document.getElementById(String(element));
+        }
+    }
+}
+loadControls()
```

Comment:
This works! I suggest calling it inside our `registerEventHandler` function, so we know that everything has been loaded _before_ we grab values from the DOM.
```suggestion
loadControls();
```